lib/internal/modules/esm/loader.js

```diff
@@ -60,6 +60,12 @@ class Loader {
     // an object with the same keys as `exports`, whose values are get/set
     // functions for the actual exported values.
     this._dynamicInstantiate = undefined;
+    // This hook is called just before the source code of an ES module file
+    // is loaded.
+    this._getSource = undefined;
+    // This hook is called just after the source code of an ES module file
+    // is loaded, but before anything is done with the string.
+    this._transformSource = undefined;
```

Comment:
Similarly, we can instantiate this to the empty function.

Reply:
thunk, not empty